components/atom/upload/src/index.js

```diff
@@ -151,9 +140,7 @@ AtomUpload.propTypes = {
   accept: PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.arrayOf(PropTypes.string)
-  ]),
-  /** Size of icon */
-  iconSize: PropTypes.oneOf(Object.values(ATOM_ICON_SIZES))
```

Comment:
loosing this feature

Reply:
its not that we are loosing it, the icon component that you pass down to the uploader can have any size, we are just decoupling the size from this component and this can be decided by the parent component.